mooncake-common/common.cmake

```diff
@@ -34,7 +39,7 @@ option(USE_CUDA "option for using gpu direct" OFF)
 option(USE_NVMEOF "option for using NVMe over Fabric" OFF)
 option(USE_TCP "option for using TCP transport" ON)
 option(USE_CXL "option for using cxl protocol" OFF)
-option(USE_ETCD "option for enable etcd as metadata server" ON)
+option(USE_ETCD "option for enable etcd as metadata server" OFF)
```

Comment:
This is more like a workaround.
I tried clang-13 can fix this issue without any code change.